### PR TITLE
[zh] sync assign-pod-node pod-scheduling-readiness taint-and-toleration

### PR DESCRIPTION
**Pull request by asa3311**

### CODE REVIEW

File: content/zh-cn/docs/concepts/scheduling-eviction/assign-pod-node.md [hunk 1]
```diff
@@ -525,6 +525,23 @@ spec.
...
+that the pod matches its own terms, and the chosen node matches all requested topologies.
+This ensures that there will not be a deadlock even if all the pods have inter-pod affinity
+specified.
+-->
+#### 调度一组具有 Pod 间亲和性的 Pod   {#scheduling-a-group-of-pods-with-inter-pod-affinity-to-themselves}
+如果当前正在调度的 Pod 是一组具有自我亲和性的 Pod 中的第一个，
+那么只要它满足其他所有的亲和性规则，它就可以被成功调度。
+这是通过以下方式确定的：确保集群中没有其他 Pod 与此 Pod 的命名空间和标签选择器匹配；
+该 Pod 满足其自定义的条件；以及选定的节点满足所有指定的拓扑要求。
+这确保即使所有的Pod都配置了Pod间的亲和性，也不会出现调度死锁的情况。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/scheduling-eviction/assign-pod-node.md:543]:
```suggestion
这确保即使所有的 Pod 都配置了 Pod 间亲和性，也不会出现调度死锁的情况。
```

File: content/zh-cn/docs/concepts/scheduling-eviction/assign-pod-node.md [hunk 2]
```diff
@@ -525,6 +525,23 @@ spec.
...
+
+If the current Pod being scheduled is the first in a series that have affinity to themselves,
+it is allowed to be scheduled if it passes all other affinity checks. This is determined by
+verifying that no other pod in the cluster matches the namespace and selector of this pod,
+that the pod matches its own terms, and the chosen node matches all requested topologies.
+This ensures that there will not be a deadlock even if all the pods have inter-pod affinity
+specified.
+-->
+#### 调度一组具有 Pod 间亲和性的 Pod   {#scheduling-a-group-of-pods-with-inter-pod-affinity-to-themselves}
+如果当前正在调度的 Pod 是一组具有自我亲和性的 Pod 中的第一个，
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/scheduling-eviction/assign-pod-node.md:539]:
```suggestion

如果当前正被调度的 Pod 在具有自我亲和性的 Pod 序列排在第一个，
```

**Review by windsonsea — COMMENTED** [review]
/approve

File: content/zh-cn/docs/concepts/scheduling-eviction/assign-pod-node.md [hunk 3]
```diff
@@ -525,6 +525,24 @@ spec.
...
+If the current Pod being scheduled is the first in a series that have affinity to themselves,
+it is allowed to be scheduled if it passes all other affinity checks. This is determined by
+verifying that no other pod in the cluster matches the namespace and selector of this pod,
+that the pod matches its own terms, and the chosen node matches all requested topologies.
+This ensures that there will not be a deadlock even if all the pods have inter-pod affinity
+specified.
+-->
+#### 调度一组具有 Pod 间亲和性的 Pod   {#scheduling-a-group-of-pods-with-inter-pod-affinity-to-themselves}
+
+如果当前正被调度的 Pod 在具有自我亲和性的 Pod 序列排在第一个，
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/assign-pod-node.md:540]:
```suggestion
如果当前正被调度的 Pod 在具有自我亲和性的 Pod 序列中排在第一个，
```

File: content/zh-cn/docs/concepts/scheduling-eviction/assign-pod-node.md [hunk 4]
```diff
@@ -525,6 +525,24 @@ spec.
...
+那么只要它满足其他所有的亲和性规则，它就可以被成功调度。
+这是通过以下方式确定的：确保集群中没有其他 Pod 与此 Pod 的命名空间和标签选择器匹配；
+该 Pod 满足其自定义的条件；以及选定的节点满足所有指定的拓扑要求。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/assign-pod-node.md:543]:
```suggestion
该 Pod 满足其自身定义的条件，并且选定的节点满足所指定的所有拓扑要求。
```

**Review by windsonsea — COMMENTED** [review]
Thanks.
/lgtm

**Pull request merged**